Model/Ens/EventHandler/EventHandlerOrder.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Copyright (c) 2020 KOUNT, INC.
+ * See COPYING.txt for license details.
+ */
+namespace Swarming\Kount\Model\Ens\EventHandler;
+
+/**
+ * Parent class for WORKFLOW events. The class contains common methods.
+ */
+class EventHandlerOrder
+{
+    /**
+     * @var \Magento\Sales\Model\OrderFactory
+     */
+    protected $orderFactory;
+
+    /**
+     * @param \Magento\Sales\Model\OrderFactory $orderFactory
+     */
+    public function __construct(
+        \Magento\Sales\Model\OrderFactory $orderFactory
```

Comment:
Can you use the `Magento\Sales\Api\OrderRepositoryInterface` and use search criteria to filter by increment_id, the functionality works as is, but repositories are better practice